Feature/scrolling

umitdincel opened the pull request.

umitdincel committed:
scrolling almost done

umitdincel committed:
remove log lines

umitdincel committed:
merge master, clean conflicts

umitdincel committed:
increase list size

umitdincel committed:
todo

yigitbey commented:
Can we add more arrows to the bottom and the top of the list? Currently it's a bit hard to see that the list can be scrolled.

Also can we show more items? Space between the list and the detail view is not aesthetically pleasing. 

The most common action is "Do nothing" on each turn, so it should be the default option. We can either move it to the top of the list, or unstage the code that makes the first item default. (Previously the last item was the default option.)

umitdincel committed:
some improvements

umitdincel committed:
arrow multiplier

umitdincel commented:
> Can we add more arrows to the bottom and the top of the list? Currently it's a bit hard to see that the list can be scrolled.

2 more added. I think so it seems better.

> Also can we show more items? Space between the list and the detail view is not aesthetically pleasing.

Updated with 10 items. You can add more items from [here](https://github.com/yigitbey/soyouhaveanidea/blob/701df0e9e20a742b6992ae06849c1fa7fd064a95/menu.py#L57)

> The most common action is "Do nothing" on each turn, so it should be the default option. We can either move it to the top of the list, or unstage the code that makes the first item default. (Previously the last item was the default option.)

I moved "Do nothing" to the top of the list. This will be a quick solution for now.
